package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "my-demo-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx playwright test --project=chromium",
+    "test-ui": "npx playwright test --ui",
+    "lint": "npx eslint . --color --fix --quiet"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@playwright/test": "^1.49.0"
+  },
+  "devDependencies": {
+    "@types/node": "^22.9.0",
+    "eslint": "^8.57.1",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-playwright": "^2.0.1",
+    "playwright": "^1.49.0",
```

Comment:
You do not need the `playwright` dependency since you already have `@playwright/test` dependency